client/data/transactions/test/resolvers.js

```diff
@@ -16,7 +16,7 @@ import {
 	updateTransactionsSummary,
 } from '../actions';
 import { getTransactions, getTransactionsSummary } from '../resolvers';
-import { getUserTimeZone } from 'wcpay/utils/test-utils';
```

Comment:
Moved `wcpay/utils/test-utils` to the test-specific directory, similar to https://github.com/Automattic/woopay/blob/8962c73aa355f52511b5f87cf04caa7a1233406d/tests/js/jest.config.js#L18